zou/app/utils/events.py

```diff
@@ -69,7 +69,14 @@ def emit(event, data={}, persist=True):
             from zou.app.stores.queue_store.job_queue import enqueue
             enqueue(func.handle_event, data)
         else:
-            func.handle_event(data)
+            try:
+                func.handle_event(data)
+            except Exception:
+                import traceback
```

Comment:
Can you handle exception this way:

```
from flask import current_app

...

current_app.logger.error(Error handling event, exc_info=1)
```

The `exc_info` parameter will include the stacktrace.